pmxbot/core.py

```diff
@@ -430,9 +430,9 @@ def merge_dicts(a, b):
 
 
 class Bot(metaclass=abc.ABCMeta):
-	def out(self, channel, s, log=True):
+	def out(self, channel, s, log=True, context=None):
 		try:
-			sent = self.allow(channel, s) and self.transmit(channel, s)
+			sent = self.allow(channel, s) and self.transmit(channel, s, context)
```

Comment:
I suspect changes here will break the signature of the other bot class in `pmxbot.irc`.